cpp/src/parquet/bloom_filter_test.cc

```diff
@@ -159,20 +152,21 @@ TEST(FPPTest, TestBloomFilter) {
 TEST(CompatibilityTest, TestBloomFilter) {
   const std::string test_string[4] = {"hello", "parquet", "bloom", "filter"};
   const std::string bloom_filter_test_binary =
-      std::string(test::get_data_dir()) + "/bloom_filter.bin";
+      std::string(test::get_data_dir()) + "/bloom_filter.xxhash.bin";
 
   PARQUET_ASSIGN_OR_THROW(auto handle,
                           ::arrow::io::ReadableFile::Open(bloom_filter_test_binary));
   PARQUET_ASSIGN_OR_THROW(int64_t size, handle->GetSize());
 
-  // 1024 bytes (bitset) + 4 bytes (hash) + 4 bytes (algorithm) + 4 bytes (length)
```

Comment:
Why remove this line? You should instead update it to correctly explain the length value.